### DISCUSSION TITLE
POST Request upload file

### DISCUSSION THREAD

**Discussion by godzail** [RapidAPI/feedback#23] [Q&A]
Goog morning,
I tried to test a post request with a file uploaded without success...
With Thunder Client No problem.
Maybe I'm doing something wrong, Is there a tutorial about sending files to test?
This is the equivalent curl call:

```
curl -X 'POST' \
  'http://127.0.0.1:8002/getimageinfo/' \
  -H 'accept: application/json' \
  -H 'Content-Type: multipart/form-data' \
  -F 'file=@test.jpg;type=image/jpeg'
```

**Comment by estruyf** [comment]
Thanks, @godzail, for opening this discussion. I'll sync this back to the team and come back at it.

**Reply by estruyf** [reply]
This issue just got fixed in version 1.4.1. The multipart body did not contain a content type, which caused an issue on the API side.

**Reply by estruyf** [reply]
Here is an example of how you can use it:

https://user-images.githubusercontent.com/2900833/183934248-cc684d58-c336-4d92-ac05-873cbd9310f4.mov

**Reply by Anag03** [reply]
Did you try to add this request to a test? 
I tried but I get an error "Missing dynamic value"
Do you know how to fix it?
